### ISSUE TITLE
extract separate static markdown verifier

### ISSUE THREAD

**Issue by kevgo** [Originate/text-runner#90]
> When writing Markdown content
I want that my changes are verified as fast as possible
So that my workflow is not slowed down and I remain productive.

#### Rules
- the static verifier checks all static content (links images) in the Markdown files
- the checks happen in parallel

#### Notes
- alternative to #73

**Comment by kevgo** [comment]
closing in favor of #96
